minard/views.py

```diff
@@ -1147,11 +1147,13 @@ def get_metric(expr, start, stop, step):
             values = get_timeseries_field('trig',field,start,stop,step)
         elif expr == 'TOTAL':
             values = get_timeseries_field('trig','TOTAL',start,stop,step)
+        elif expr == 'polling':
+            values = get_timeseries_field('trig','polling',start,stop,step)
         else:
             values = get_timeseries(expr,start,stop,step)
 
         interval = get_interval(step)
-        if expr in TRIGGER_NAMES or expr in ('TOTAL','L1','L2','ORPHANS','BURSTS'):
+        if expr in TRIGGER_NAMES or expr in ('TOTAL','L1','L2','ORPHANS','BURSTS', 'polling'):
```

Comment:
I don't think this is correct. The way you have the polling marked in minard-dispatch it is treated as a binary thing (the last event in each second is what is saved for the bucket), not a count, so it's not correct to divide by the interval. You can have a look at how the gtid is treated for an example where the "last" value in a bin is what is shown on the timeseries.

Alternatively, if you want to treat the polling bit like the other trigger words, I think you'll have to rework the minard-dispatch script so that it increments the field for every event instead of just setting the last event. The advantage to this method is that for longer timescales you can actually see what fraction of events were polling events whereas with the current method it will always show the last event in each bucket (so for longer timescales, it won't really represent anything interpretable).

Reply:
Ah yah, sorry. I haven't figured out a good way to test this locally yet.